terraform/container-definitions/web_container_definition.json

```diff
@@ -132,11 +132,11 @@
       },
       {
         "name": "AUDIT_VACANCIES_WORKSHEET_GID",
```

Comment:
The GIDs actually I don't think _need_ to be in the web container definition, so they can probably be binned

Reply:
I've removed the GIDs from the web container definition 👍